plugins/frontend/backstage-plugin-bugsnag/config.d.ts

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright 2022 Larder Software Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+export interface Config {
+  /** Results per page for bugsnag plugin */
+  bugsnag?: {
+    /**
+     *
+     * @visibility frontend
+     */
+    resultsPerPage: number;
```

Comment:
Does this need to be optional as well or is it enough if `bugsnag` above is opt?

Reply:
Yeah, I think we also need to make it optional, at least by the pattern I see in other schemas